### DISCUSSION TITLE
Role of "hi-from-lambda.js" in functions folder

### DISCUSSION THREAD

**Discussion by iamsantanubanerjee** [thuliteio/doks#684] [Q&A] [answered]
What is the above mentioned file needed for? Can it be safely removed?

Additional query: Can the "docs-versions.yml" be removed without significant consequences as well?

Thank you in advance.

**Comment by h-enk** [comment, marked as answer]
> What is the above mentioned file needed for? Can it be safely removed?

Just an example — it can safely be removed

> Additional query: Can the "docs-versions.yml" be removed without significant consequences as well?

If you don't use versioning, it can safely be removed
